src/hooks/useSubscriptions.ts

```diff
@@ -144,9 +141,7 @@ export function useProposalSubscription(
       onProposalHandler
     )
 
-    return () => {
-      proposalSubscription.current = null
-    }
+    return () => proposalSubscription.current.unsubscribe()
```

Comment:
This should be:

```suggestion
    return () => {
      proposalSubscription.current.unsubscribe()
    }
```

because useEffect expects undefined during unsubscribing.